activemodel/lib/active_model/serialization.rb

```diff
@@ -69,6 +69,37 @@ module ActiveModel
   #   person.serializable_hash(only: 'name')
   #   person.serializable_hash(include: :address)
   #   person.serializable_hash(include: { address: { only: 'city' }})
+  #
+  # Example with <tt>:include</tt> option
+  #
+  #   class User
+  #     include ActiveModel::Serializers::JSON
+  #     attr_accessor :name, :notes # Emulate has_many :notes
+  #     def attributes
+  #       {'name' => nil}
+  #     end
+  #   end
+  #
+  #   class Note
+  #     include ActiveModel::Serializers::JSON
+  #     attr_accessor :title, :text
+  #     def attributes
+  #       {'title' => nil, 'text' => nil}
+  #     end
+  #   end
+  #
+  #   note = Note.new
+  #   note.title = 'Battle of Austerlitz'
+  #   note.text = 'Some text here'
+  #
+  #   user = User.new
+  #   user.name = 'Napoleon'
+  #   user.notes = [note]
+  #
+  #   user.serializable_hash
+  #   #=> {"name" => "Napoleon"} 
+  #   user.serializable_hash(include: {notes: {only: 'title'}})
```

Comment:
Fixed!  [In the same manner like here](https://github.com/rusikf/rails/blob/patch-1/activemodel/lib/active_model/serialization.rb#L71)